BaseApp/COD3031.txt

```diff
@@ -0,0 +1,125 @@
+OBJECT Codeunit 3031 DotNet_SmtpMessage
+{
+  OBJECT-PROPERTIES
+  {
+    Date=;
+    Time=;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetSmtpMessage@1000 : DotNet "'Microsoft.Dynamics.Nav.SMTP, Version=13.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.Microsoft.Dynamics.Nav.SMTP.SmtpMessage";
+
+    [External]
+    PROCEDURE CreateMessage@1();
+    BEGIN
+      DotNetSmtpMessage := DotNetSmtpMessage.SmtpMessage;
+    END;
+
+    [External]
+    PROCEDURE SetFromAddress@2(FromAddress@1000 : Text);
+    BEGIN
+      DotNetSmtpMessage.FromAddress := FromAddress;
+    END;
+
+    [External]
+    PROCEDURE SetFromName@3(FromName@1000 : Text);
+    BEGIN
+      DotNetSmtpMessage.FromAddress := FromName;
+    END;
+
+    [External]
+    PROCEDURE SetToAddress@4(ToAddress@1000 : Text);
+    BEGIN
+      DotNetSmtpMessage."To" := ToAddress;
+    END;
+
+    [External]
+    PROCEDURE SetSubject@5(Subject@1000 : Text);
+    BEGIN
+      DotNetSmtpMessage.Subject := Subject;
+    END;
+
+    [External]
+    PROCEDURE SetAsHtmlFormatted@6(HtmlFormatted@1000 : Boolean);
+    BEGIN
+      DotNetSmtpMessage.HtmlFormatted := HtmlFormatted;
+    END;
+
+    [External]
+    PROCEDURE SetTimeout@7(Timeout@1000 : Integer);
+    BEGIN
+      DotNetSmtpMessage.Timeout := Timeout;
+      DotNetSmtpMessage.ConvertBase64ImagesToContentId;

```

Comment:
Is this line supposed to be there?

Reply:
Oh. Yes it shouldn't be here :) Well you what happens with copy-paste.

Reply:
:-D I'll remove it and move forward with this request. Thanks!